src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/DiagnosticCounter.cs

```diff
@@ -20,6 +20,11 @@ namespace System.Diagnostics.Tracing
     /// DiagnosticCounter is an abstract class that serves as the parent class for various Counter* classes,
     /// namely EventCounter, PollingCounter, IncrementingEventCounter, and IncrementingPollingCounter.
     /// </summary>
+#if !ES_BUILD_STANDALONE
+#if !FEATURE_WASM_PERFTRACING
+    [System.Runtime.Versioning.UnsupportedOSPlatform("browser")]
+#endif
+#endif
```

Comment:
nit: this one and others:

```suggestion
#if !ES_BUILD_STANDALONE && !FEATURE_WASM_PERFTRACING
    [System.Runtime.Versioning.UnsupportedOSPlatform("browser")]
#endif
```

Reply:
This was intentional to make it easier to grep for occurrences of `#if !FEATURE_WASM_PERFTRACING`

Reply:
Do we have any principle for this? I would prefer to not loose syntactic sugar for the sake of grepping.